code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -128,14 +128,18 @@ GLOBAL_LIST_INIT(metal_recipes, list(
 	point_value = 2
 	table_type = /obj/structure/table
 
+/obj/item/stack/sheet/metal/examine(mob/user)
+	. = ..()
+	. += "<span class='notice'>Metal is used in various different construction sequences. It can be used on lattices to create a floor tile.</span>"
```

Comment:
I don't think you can use metal sheets on lattices to make floor tiles. You need an actual floor tile item.

Reply:
you're right, this slipped through the cracks